src/thunderbots/software/ai/hl/stp/action/chip_action.h

```diff
@@ -20,6 +21,7 @@ class ChipAction : public Action
      * Returns the next Intent this ChipAction wants to run, given the parameters.
      *
      * @param robot The robot that should perform the chip
+     * @param ball The ball being kicked
      * @param chip_origin The location where the chip will be taken
```

Comment:
We should clarify what the `chip_location` is here? How is it used in relation to the `ball`? I know it's a bit of an implementation detail, but I think that one should understand what each is used for in order to use this class well.

Reply:
fixed